digital-card-default.properties

```diff
@@ -120,3 +120,5 @@ mosip.iam.adapter.renewal-before-expiry-interval=1440
 mosip.iam.adapter.self-token-renewal-enable=true
 mosip.auth.filter_disable=false
 auth.server.admin.allowed.audience=mosip-admin-client,mosip-resident-client
+
+mosip.digitalcard.service.template={RPR_UIN_CARD_TEMPLATE:uin_card_template.properties;vid-card-type:vid_card_template.properties}
```

Comment:
Add detailed comment with example on the property value's format.

Reply:
taken care